src/Mvc/Mvc.Testing/ref/Microsoft.AspNetCore.Mvc.Testing.netcoreapp3.0.cs

```diff
@@ -36,12 +36,6 @@ public partial class WebApplicationFactory<TEntryPoint> : System.IDisposable whe
         protected virtual Microsoft.Extensions.Hosting.IHost CreateHost(Microsoft.Extensions.Hosting.IHostBuilder builder) { throw null; }
         protected virtual Microsoft.Extensions.Hosting.IHostBuilder CreateHostBuilder() { throw null; }
         protected virtual Microsoft.AspNetCore.TestHost.TestServer CreateServer(Microsoft.AspNetCore.Hosting.IWebHostBuilder builder) { throw null; }
-        protected virtual Microsoft.AspNetCore.Hosting.IWebHostBuilder CreateWebHostBuilder() { throw null; }
```

Comment:
This is causing compiler errors. Looks like a bad rebase.

Reply:
Arthur is OOF right now. I'm going to update this PR and see if we can get it merged today.